### ISSUE TITLE
Allow passing in specific alpha into adj_alpha

### ISSUE THREAD

**Issue by mattwigway** [pysal/mgwr#80]
Currently, GWRResults is [hard-wired]() to return the adjusted alpha for .1, .05, and .001. It would be great to allow the user to specify this.

**Comment by TaylorOshan** [comment]
Good call! Should be easy enough to make the default input parameter equal to a list of values rather than hard coding it so that the user can easily change it as needed.
